packages/backend-modules/payments/graphql/resolvers/_mutations/createCheckoutSession.ts

```diff
@@ -1,6 +1,10 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { GraphqlContext } from '@orbiting/backend-modules-types'
-import { Shop, Offers, utils } from '../../../lib/shop'
+import {
+  Shop,
+  Offers,
+  checkIntroductoryOfferEligieblity,
```

Comment:
There's an "e" in the function name that shouldn't be there: checkIntroductoryOffer**Eligibility**